README.md

```diff
@@ -4,19 +4,19 @@ This is a React module capstone project. It is built with react and redux by usi
 This application shows the statistics of covid cases in various countries.
 ## Screenshot
 
-![screenshot]()
+![screenshot](https://user-images.githubusercontent.com/38283436/159509800-f54c6d0c-5a1a-4126-8148-c0420d317c64.png)
 
 ## Live Demo
 
-[Live Link]()
+[Live Link](https://sirriryisa.github.io/covid-19-data/)
```

Comment:
- [ ] This link should be pointing to the application deployed using Heroku, Netlify, or GitHub Pages. Now it is pointing to your Readme file.